test/css/default.css

```diff
@@ -3859,6 +3859,326 @@ table {
     table.responsive tr {
       display: block; } }
 
+body,
+html {
+  font-size: 16px; }
+
+a,
+a:visited,
+a:link,
+a.link:visited,
+.alink:visited {
+  color: inherit; }
+
+.text-large {
+  font-size: 1.25em; }
+
+h1.fancy,
+h2.fancy,
+h3.fancy,
+h4.fancy,
+h5.fancy,
+h6.fancy {
+  font-style: italic; }
```

Comment:
Kan je dit niet beter italic noemen oblique ofzoo

Reply:
Goed punt. Ga ik doen.. in m'n issues fix.